### ISSUE TITLE
Piracy is added to non-piracy issues

### ISSUE THREAD

**Issue by jackthehack21** [jlippold/tweakCompatible#23131]
The bot seems to think a few tweaks are piracy tweaks but in fact theyre not a example is #22987 where Fluo was marked as piracy but is a theme, not a piracy tweak.

**Comment by EndlessExtis** [comment]
This true for NoPageDots#21799 And Eclipse X#21782 both of which are themes that change the homescreen

**Comment by jlippold** [comment]
I’ve gotten requests from the r/jailbreak mods to make the app more restrictive against piracy.  Do you mind checking if the package Id, or the repo name matches any of the values in this file? https://github.com/jlippold/tweakCompatible/blob/master/docs/bans.json

**Comment by jackthehack21** [comment]
Yes it matches repo but doesn’t necessarily mean their all piracy tweaks.

**Comment by jlippold** [comment]
Which repo?

**Comment by Jackthehaxk21** [comment]
Xarold sorry don’t know there is a original so to me it was the first release so I defualt thought it was the original by Xarold, sorry
But one more question @jlippold when I do submit stuff it says IPhone 7,2 but I have a iPhone 6
